test/mocks/authorize_net_mock.exs

```diff
@@ -1,6 +1,61 @@
-  defmodule Gringotts.Gateways.AtuhtorizeNetMock do
+  defmodule Gringotts.Gateways.AuthorizeNetMock do
 
-    def successful_response do
-      
+    def successful_purchase_response do
+      {:ok,
+      %HTTPoison.Response{body: "﻿<?xml version=\"1.0\" encoding=\"utf-8\"?><createTransactionResponse xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\" xmlns=\"AnetApi/xml/v1/schema/AnetApiSchema.xsd\"><refId>123456</refId><messages><resultCode>Ok</resultCode><message><code>I00001</code><text>Successful.</text></message></messages><transactionResponse><responseCode>1</responseCode><authCode>C7HPT1</authCode><avsResultCode>Y</avsResultCode><cvvResultCode>P</cvvResultCode><cavvResultCode>2</cavvResultCode><transId>60036553096</transId><refTransID /><transHash>5D6782A03246EE3BAFABE8006E32DE97</transHash><testRequest>0</testRequest><accountNumber>XXXX0015</accountNumber><accountType>MasterCard</accountType><messages><message><code>1</code><description>This transaction has been approved.</description></message></messages><transHashSha2 /></transactionResponse></createTransactionResponse>",
+       headers: [{"Cache-Control", "private"},
```

Comment:
Actually, I kept the body in one line just so that I am able to see the responses properly, otherwise they are so big I have to keep scrolling to see even a single complete response.